data/changelogs/pages.yaml

```
@@ -0,0 +1,86 @@
+---
+link: "/pages/platform/changelog/"
+productName: Pages
+entries:
+- publish_date: '2023-02-14'
+  title: Analytics Engine now available in Functions
+  description: Added support for [Analytics Engine](/analytics/analytics-engine/)
+    in Functions.
+- publish_date: '2023-01-05'
+  title: Queues now available in Functions
+  description: Added support for [Queues](/queues/) producer in Functions.
+- publish_date: '2022-12-15'
+  title: API messaging update
+  description: Updated all API messaging to be more helpful.
+- publish_date: '2022-12-01'
+  title: Ability to delete aliased deployments
+  description: |-
+    Aliased deployments can now be deleted.
+
+    If using the API, you will need to add the query parameter `force=true`.
+- publish_date: '2022-11-19'
+  title: Deep linking to a Pages deployment
+  description: |-
+    You can now deep-link to a Pages deployment in the dashboard with `:pages-deployment`.
+
+    An example would be `https://dash.cloudflare.com?to=/:account/pages/view/:pages-project/:pages-deployment`.
+- publish_date: '2022-11-17'
+  title: Functions GA and other updates
+  description: |-
+    Pages functions are now GA. For more information, refer to the [blog post](https://blog.cloudflare.com/pages-function-goes-ga/).
+
+    We also made the following updates to Functions:
+
+    - [Functions metrics](https://dash.cloudflare.com?to=/:account/pages/view/:pages-project/analytics/production) are now available in the dashboard.
+    - [Functions billing](/pages/platform/functions/pricing) is now available.
+    - The [Unbound usage model](/workers/platform/limits/#response-limits) is now available for Functions.
+    - [Secrets](/pages/platform/functions/bindings/#secrets) are now available.
+    - Functions tailing is now available via the [dashboard](https://dash.cloudflare.com?to=/:account/pages/view/:pages-project/:pages-deployment/functions) or with Wrangler (`wrangler pages deployment tail`).
+- publish_date: '2022-11-15'
+  title: Service bindings now available in Functions
+  description: Service bindings are now available in Functions. For more details,
+    refer to the [docs](/pages/platform/functions/bindings/#service-bindings).
+- publish_date: '2022-11-03'
+  title: Ansi color codes in build logs
+  description: Build log now supports ansi color codes.
+- publish_date: '2022-10-05'
+  title: Deep linking to a Pages project
+  description: You can now deep-link to a Pages project in the dashboard with `:pages-project`.
+    An example would be `https://dash.cloudflare.com?to=/:account/pages/view/:pages-project`.
```

```suggestion
  description: |-
    You can now deep-link to a Pages project in the dashboard with `:pages-project`.

    An example would be `https://dash.cloudflare.com?to=/:account/pages/view/:pages-project`.
```
We were losing some of the inline code formatting (no background color) for some reason. Making it into two paragraphs, like in the "Deep linking to a Pages deployment" changelog entry (2022-11-19), should fix it.